presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -69,6 +69,13 @@ Array Functions
 
     Returns the minimum value of input array.
 
+.. function:: array_normalize(x, p) -> array
+
+   Normalizes array ``x`` by dividing each element by the p-norm of the array.
+   It is equivalent to `TRANSFORM(array, v -> v / REDUCE(array, 0, (a, v) -> a + POW(ABS(v), p), a -> POW(a, 1 / p))`,
```

Comment:
Use two quotes on each side: \`\`TRANS...\`\`. Otherwise, it will be italic font than math ones. To confirm if it works, compile presto-docs locally and open the file in your target dir: `<YOUR_DIR>/presto/presto-docs/target/html/functions/array.html`